### PR TITLE
Installation docs

### PR DESCRIPTION
**Pull request by StillAbeginnerr**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,13 +4,43 @@
...
+
+Type The following Commands and Press Enter: 
+
+npm install yarn
+
+yarn install
+
+node --inspect -r ts-node/register src/index.ts
+
+Now , Scan QR code from WhatsApp Mobile.
```
Comment (Meghdut-Mandal) [inline README.md:31]: Remove the word now

File: README.md [hunk 2]
```diff
@@ -4,13 +4,43 @@
...
+
+You can now easily use the bot commands in your whatsapp mobile.
```
Comment (Meghdut-Mandal) [inline README.md:33]: This portion should be placed outside of the Steps

**Comment by Meghdut-Mandal** [comment]
Remove the deploy to Heroku Button and section

**Comment by StillAbeginnerr** [comment]
Okay , I shall do these changes.

**Pull request merged**